### DISCUSSION TITLE
Size limit on attachment field

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#8860] [Help] [answered]
How would you code a size limite error message ?
I manage to get the file size using {{ Form.Fields.myAttach.0.size }} 
I was trying with the validation rules of my attachment field but not working :
<img width="471" alt="image" src="https://user-images.githubusercontent.com/28592069/204594266-d99f5c03-037e-46d2-944d-5f8025f2fff5.png">
@mielune

**Comment by melohagan** [comment, marked as answer]
Hey @NicolasFabre81 

This is currently not possible. I have raised an issue here: https://github.com/Budibase/budibase/issues/8936
